src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsassessforearlyreleaseapi/resource/OffenderResource.kt

```diff
@@ -171,4 +179,98 @@ class OffenderResource(private val offenderService: OffenderService) {
 
     offenderService.optOut(prisonNumber)
   }
+
+  @PostMapping("/offender/{prisonNumber}/current-assessment/standard-address-check-request")
```

Comment:
Can we put these on an address controller? 

This this controller will get massive otherwise